src/app/shared/app-details-description/app-details-description.component.ts

```diff
@@ -45,6 +45,22 @@ export class AppDetailsDescriptionComponent implements OnInit {
         this.showThumbnails =
           this.items && this.items.length > 1 && !this.isHandset;
       }
+
+      this.meta.addTags([
+        { name: 'twitter:card', content: 'summary_large_image' },
+        { name: 'og:description', content: this.app.summary },
+        { name: 'og:title', content: this.app.name },
+      ]);
+
+      if (
+        this.app.screenshots &&
+        this.app.screenshots.length > 0 &&
+        this.app.screenshots[0].imgDesktopUrl
+      ) {
+        this.meta.addTags([
+          { name: 'og:image', content: this.app.screenshots[0].imgDesktopUrl },
```

Comment:
We could use the app icon instead, I'm not sure what we want here. If we go for the first screenshot, we should have a fallback to the icon.